Python/Product/Common/Strings.resx

```diff
@@ -2257,28 +2257,28 @@ Expand selection to the full expression?</value>
     <value>Debugger package could not be loaded</value>
   </data>
   <data name="PtvsdDisableCaption" xml:space="preserve">
-    <value>Disable the experimental debugger</value>
+    <value>Enable the legacy debugger</value>
   </data>
   <data name="PtvsdDisableSubtext" xml:space="preserve">
-    <value>Future debugging sessions will not use the experimental debugger. You can also change this setting in the Tools/Options/Python/Experimental page.</value>
+    <value>Future debugging sessions will use the legacy debugger. You can change this setting back in Tools &gt; Options &gt; Python &gt; Debugging page.</value>
   </data>
   <data name="PtvsdLearnMoreCaption" xml:space="preserve">
     <value>Learn more</value>
   </data>
   <data name="PtvsdLearnMoreSubtext" xml:space="preserve">
-    <value>View documentation in your web browser on supported Python environments and how to upgrade or uninstall the ptvsd debugger package.</value>
+    <value>View documentation on supported Python environments and how to upgrade or uninstall the ptvsd debugger package.</value>
   </data>
   <data name="PtvsdIncompatibleEnvTitle" xml:space="preserve">
-    <value>Debugger is incompatible with this environment</value>
+    <value>Debugger does not support this Python environment</value>
   </data>
   <data name="PtvsdIncompatibleEnvMessage" xml:space="preserve">
-    <value>The experimental debugger is incompatible with this Python environment. See the documentation for a list of supported Python environments.</value>
+    <value>To debug this Python environment use the legacy debugger instead. It is likely using an older version of Python, see the documentation for a list of supported Python versions and environment types.</value>
```

Comment:
"To debug this Python environment, please use the legacy debugger. Older versions of Python are not supported. See our documentation ..."

(Also, if this is going as the content in a task dialog, we can use a hyperlink in the message - `<a href="...">the documentation</a>`